docs/core/porting/third-party-deps.md

```diff
@@ -2,22 +2,34 @@
 title: Analyze dependencies to port code to .NET Core
 description: Learn how to analyze external dependencies in order to port your project from .NET Framework to .NET Core.
 author: cartermp
-ms.date: 12/07/2018
+ms.date: 09/23/2019
 ms.custom: seodec18
 ---
 # Analyze your dependencies to port code to .NET Core
 
-To port your code to .NET Core or .NET Standard, you must understand your dependencies. External dependencies are the [NuGet packages](#analyze-referenced-nuget-packages-in-your-projects) or [DLLs](#analyze-dependencies-that-arent-nuget-packages) you reference in your project, but that you don't build. Evaluate each dependency and develop a contingency plan for the ones that aren't compatible with .NET Core. Here's how to determine if a dependency is compatible with .NET Core.
+To port your code to .NET Core or .NET Standard, you must understand your dependencies. External dependencies are the [NuGet packages](#analyze-referenced-nuget-packages-in-your-projects) or [DLLs](#analyze-dependencies-that-arent-nuget-packages) you reference in your project, but that you don't build yourself.
 
-## Analyze referenced NuGet packages in your projects
+## Migrate your packages to `PackageReference`
 
-If you reference NuGet packages in your project, you need to verify if they're compatible with .NET Core.
-There are two ways to accomplish that:
+.NET Core uses [PackageReference](/nuget/consume-packages/package-references-in-project-files) to specify package dependencies. If you are using [packages.config](/nuget/reference/packages-config) to specify your packages, you will need to convert over to `PackageReference` because `packages.config` is unsupported for .NET Core.
 
-* [Using the NuGet Package Explorer app](#analyze-nuget-packages-using-nuget-package-explorer)
-* [Using the nuget.org site](#analyze-nuget-packages-using-nugetorg)
+You can learn how to migrate at [Migrate from packages.config to PackageReference](/nuget/reference/migrate-packages-config-to-package-reference).
 
-After analyzing the packages, if they're not compatible with .NET Core and only target .NET Framework, you can check if the [.NET Framework compatibility mode](#net-framework-compatibility-mode) can help with your porting process.
+## Upgrade your NuGet packages after converting them
+
+After converting to `PackageReference`, you need to verify if your packages are compatible with .NET Core.
+
+First, upgrade your packages to the latest version that you can. It's likely that newer versions of your package dependencies are already compatible with .NET Core.
```

Comment:
do we need to give instructions on how to do this?

Reply:
I'll mention VS